content/departments/cloud/technical-docs/oidc_site_admin.md

```diff
@@ -124,6 +124,26 @@ Run the following SQL query for security event logs:
 SELECT * FROM security_event_logs WHERE argument @> '{"sourcegraph_operator": true}';
 ```
 
+### How to enable/disable SOAP for a managed instance?
+
+Go to the `$CUSTOMER` direcotry.
+
+For v1 instances:
+
+1. Remove or add the following line in the `config.yaml` file to enable or disable SOAP respectively:
+
+   ```yaml
+   disableSourcegraphManagementAccess: true
+   ```
+
+1. Update the GSM value of the secret `CLOUD_SITE_CONFIG` by running `mi sync cloud-site-config`.
+1. Pull down the new GSM value to the VM by running `terraform apply` or trigger an apply on the Terraform Cloud.
+1. [Reload the manage instance](https://github.com/sourcegraph/deploy-sourcegraph-managed/actions/workflows/reload_instance.yml).
+
+For v2 instances:
+
+1. TODO
```

Comment:
v2 doesn't have a way to disable SOAP access yet, so maybe leave it empty for now?

```suggestion
<!-- TODO(@michaellzc): update docs after it is implemented -->

Disabling SOAP is not supported on v2
```

Reply:
Sounds good, thanks!